pkg/cli/run.go

```diff
@@ -54,6 +54,11 @@ func run(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	err = checkMutuallyExclusiveFlags(cmd)
```

Comment:
this could also be done in PreRunE but it's fine here.

Reply:
Done! Thank you!